clang/test/SemaCXX/verbose-trap.cpp

```diff
@@ -45,3 +48,10 @@ void f2() {
 void test() {
   f<constCat3, constMsg3>(nullptr);
 }
+
+/// Arguments must be null terminated.
+constexpr char K[] = {'a', 'b'};
```

Comment:
Can we try a case where the array is null terminated. How about a case where the array has multiple null terminators etc

Reply:
I added a null-terminated array test case, but I can't check that we really have the right string here.